### PR TITLE
Margin between buttons and picture

### PR DESCRIPTION
**Pull request by Aakash-DD**
Added some spacing between buttons and between a button and a image for mobile view ,basically to be responsive.

### CODE REVIEW

**Comment by mariobehling** [comment]
Please open an issue and provide screenshot and test page first. Thanks!

**Comment by mariobehling** [comment]
Oh sorry. I see you referenced it in the title. Please reference it in the description in future and provide a test link.

File: index.html [hunk 1]
```diff
@@ -1342,7 +1342,7 @@ <h1>UNESCO Hackathon on Saturday March 24/Sunday March 25</h1>
 
                         <a href="https://unesco.sciencehack.asia" class="btn" target="default">
                             </i>Hackathon Website</a>
-                        <a href="https://eventyay.com/e/db15e7db/" class="btn btn-hollow-red" target="default">Tickets</a>
+                        <a href="https://eventyay.com/e/db15e7db/" class="btn btn-hollow-red" target="default" style="margin-bottom:4px;">Tickets</a>
```
Comment (simsausaurabh) [inline index.html]: Avoid inline styles.

File: index.html [hunk 2]
```diff
@@ -1316,7 +1316,7 @@ <h1>50+ Tech Companies and Projects at the FOSSASIA Exhibition</h1>
                                         CentOS, FreeBSD and many more at the exhibition.</p>
                                     <p class="lead">Opening on Thursday, March 22 at 12pm with a reception, running daily from 10am - 6pm
                                         until Saturday evening (March 24). Limited Free Hall passes available.</p>
-                                    <a href="./tickets/" class="btn" target="default">
+                                    <a href="./tickets/" class="btn" target="default" style="margin-top: 4px; margin-bottom: 4px;margin-right: 4px;">
```
Comment (simsausaurabh) [inline index.html]: Avoid inline styles.

**Pull request merged**